CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## HEAD
+
+- Enhancements
+    - [connection] Adding support to env configurations to tests
+    - [query] Adding support to empty map or keyword fields in `Query.Builder.where/2`
```

Comment:
All current changelog entries do not use any style of "tagging" for their entries. Would it be possible to reword the new entries to match the file?